internal/db/egress/dynamodb.go

```diff
@@ -35,13 +35,14 @@ type egressRecord struct {
 	// This allows sorting by time within each date partition
 	SK string `dynamodbav:"SK"`
 
+	Invocation string `dynamodbav:"invocation"`
 	NodeID     string `dynamodbav:"nodeID"`
 	Receipts   string `dynamodbav:"receipts"`
 	Endpoint   string `dynamodbav:"endpoint"`
 	ReceivedAt string `dynamodbav:"receivedAt"`
 }
 
-func newRecord(nodeID did.DID, receipts ucan.Link, endpoint *url.URL) egressRecord {
+func newRecord(invocation ucan.Link, nodeID did.DID, receipts ucan.Link, endpoint *url.URL) egressRecord {
```

Comment:
What happens if we submit the same receipt batch multiple times? Don't we need to check that we haven't already recorded the batch?